packages/kafka/kafka.spec

```diff
@@ -107,5 +107,8 @@ rm -rf %{buildroot}
 
 
 %changelog
+* Wed Nov 11 2020 "Brandon Dunne" <bdunne@redhat.com>
```

Comment:
Can you add \<version\>-\<release\> `2.3.1-2` at the end of this line?

Reply:
:+1: Done